pytorch_lightning/loops/fit_loop.py

```diff
@@ -195,6 +197,22 @@ def on_run_start(self) -> None:  # type: ignore[override]
         """Calls the ``on_train_start`` hook."""
         # reset train dataloader and val dataloader
         self.trainer.reset_train_val_dataloaders(self.trainer.lightning_module)
+
+        # global_step is incremented during checkpointing (#11555)
```

Comment:
Shouldn't this comment be in L207 instead?

Reply:
yes!